fetchai/ledger/api/contracts.py

```diff
@@ -70,7 +71,7 @@ def action(self, contract_digest: Address, contract_owner: Address, action: str,
                *args, shard_mask: BitVector = None):
         # Default to wildcard shard mask if none supplied
         if not shard_mask:
-            print("WARNING: defaulting to wildcard shard mask as none supplied")
+            logging.warn("Defaulting to wildcard shard mask as none supplied")
```

Comment:
`logging.warn` is deprecated, use `logging.warning` instead.
```suggestion
            logging.warning("Defaulting to wildcard shard mask as none supplied")
```

Reply:
Done.